protocol/contracts/farm/facets/SiloFacet/UpdateSilo.sol

```diff
@@ -104,4 +105,18 @@ contract UpdateSilo is SiloExit {
             s.a[account].lastRain = 0;
         }
     }
+
+    function convertSeeds(address account) public {
```

Comment:
I'm going to keep it public for now, unless I'm missing something big, because 1. they are functions being used in the test, and changing them to internal would require making a new function in a mock, and 2. since there is no gas advantage from the switch, and since people would never be incentivized to call this function alone, keeping it public shouldn't matter too much.